src/components/Renderer/index.tsx

```diff
@@ -70,6 +66,40 @@ export const getWidget = (
 	return typeWidgets[valueType] ?? typeWidgets.default;
 };
 
+export interface WidgetProps {
+	value?: Value;
+	schema?: JSONSchema;
+	extraContext?: object | undefined;
+	uiSchema?: UiSchema;
+	uiSchemaWidget?: UiSchema['ui:widget'];
+	extraFormats?: Format[];
+}
+
+export const Widget = ({
+	value,
+	schema,
+	extraFormats,
+	...otherProps
+}: WidgetProps) => {
+	const processedValue = getValue(value, schema);
+
+	if (processedValue === undefined || processedValue === null || !schema) {
```

Comment:
Let's simplify it a bit:
```suggestion
	if (processedValue == null || !schema) {
```